development/releasing.md

```diff
@@ -23,16 +23,16 @@ Once all issues have passed acceptance testing and have been merged into `master
 @core_devs I've just created the `<major>.<minor>.x` release branch. Please be aware that any further changes intended for this release will have to be merged to `master` then backported. Thanks!
   ```
 2. Build a beta named `<major>.<minor>.<patch>-beta.1` by pushing a git tag and when CI completes successfully notify the QA team that it's ready for release testing.
-3. [Import translations keys](translations.md#adding-new-keys) into POE and notify the #translations Slack channel translate new and updated values, for example:
+3. Get any new phrases ready for the translations team. Use [the CLI](https://github.com/medic/cht-core/tree/master/scripts/poe) to upload `messages-en.properties`. Then notify the #translations Slack channel translate new and updated values, for example:
   ```
-@channel I've just updated the translations in POE. These keys have been added: "<added-list>", and these keys have been updated: "<updated-list>"
+@channel X.Y.Z has entered release testing, and I have updated the POE editor with new and changed translations, ready to be translated!"
```

Comment:
I changed this because we don't get those values out of the tool when you use it, and I presumed we weren't manually compiling that list, because when you use the translation tool it should tell you what is and isn't translated, correct?

Also, my understanding is that we only add new keys manually in code, all updates should be done in the translation tool?

Reply:
Yes you can see new items there.

No updated values should be entered directly in to messages-en.properties and then uploaded using the tool: https://github.com/medic/medic-docs/blob/master/development/translations.md#modifying-any-existing-translation-values

The way to build the list of what's changed is to diff messages-en.properties against the previous release. I think we need to keep this (even though nobody is actually running translations right now).